### PR TITLE
Poetickz excercise 05

### PR DESCRIPTION
**Pull request by Poetickz**
This PR:
-Created SumWithOutMaxMin class.
-Created a test file with all methods of the SumWithOutMaxMin class.
-Tested all methods of SumWithOutMaxMin class.

### CODE REVIEW

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+class SumWithOutMaxMin
+    
+    def self.sort_array_of_intergers(array_of_intergers)
```
Comment (JahlHernandez) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]: You could probably remove this method, and just call `array_of_integers.sort` from `eliminate_min_max` instead.

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        array_of_intergers.sort
+    end
+
+    def self.eliminate_min_and_max(array_of_intergers)
```
Comment (JahlHernandez) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]: This is mostly nitpicking, but I'd argue that `remove_min_and_max` makes more sense in the context of what you're doing to the array.

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/spec/sum_with_out_max_min_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require 'rspec/autorun'
+require '../sum_with_out_max_min.rb'
+
+describe SumWithOutMaxMin do
+
+    describe ".sort_array_of_intergers" do
+        it "should return a sorted array from smallest to largest value" do
```
Comment (JahlHernandez) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/spec/sum_with_out_max_min_spec.rb]: Another possible description could be `it "should return an array sorted on ascending order"`

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/spec/sum_with_out_max_min_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+require 'rspec/autorun'
+require '../sum_with_out_max_min.rb'
+
+describe SumWithOutMaxMin do
+
+    describe ".sort_array_of_intergers" do
+        it "should return an array sorted on ascending order" do
```
Comment (HectorMg) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/spec/sum_with_out_max_min_spec.rb]: Grammar: "sorted **in** ascending order"

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+
+class SumWithOutMaxMin
+
```
Comment (HectorMg) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]: I'd personally remove one of these blank lines :)

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+class SumWithOutMaxMin
+    
+   
+    def self.remove_min_and_max(array_of_intergers)
+        sorted_array = array_of_integers.sort
+        sorted_array.shift
```
Comment (HectorMg) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]: Ideally, we avoid using methods that directly mutate an existing variable/object/data structure. First you called it `sorted_array` but by the time you returned it you executed 2 other operations on it, making the initial variable name less relevant and possibly confusing anyone who comes in contact with this code.
```suggestion
        shifted_array = sorted_array.shift
```

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+        sorted_array.pop
```
Comment (HectorMg) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]:
```suggestion
         array_without_max_and_min = shifted_array.pop
```

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+        return sorted_array
```
Comment (HectorMg) [inline ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]:
```suggestion
        return array_without_max_and_min
```

File: ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb (hunk 6 above)
Comment (Poetickz) [reply ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb]: If I assign the array in that way, the variable will  be the poped or shifted value not the array.

**Pull request merged**